web/js/map/ui.js

```diff
@@ -80,13 +80,13 @@ export function mapui(models, config) {
       .on('remove', removeLayer)
       .on('visibility', updateLayerVisibilities)
       .on('opacity', updateOpacity)
-      .on('update', updateLayerOrder);
+      .on('update', updateLayer);
     models.date.events.on('select', updateDate);
     models.palettes.events
-      .on('set-custom', updateLookup)
-      .on('clear-custom', updateLookup)
-      .on('range', updateLookup)
-      .on('update', updateLookup);
+      .on('set-custom', updateLayer)
+      .on('clear-custom', updateLayer)
```

Comment:
Why did you change which function is passed as a callback on these four lines?

Reply:
When we found out that `cache.removeWhere` was broken, I removed it from `updateLookup`. This only left a call to `reloadLayers` within `updateLookup`. But there was another function only calling `reloadLayers`, `updateLayerOrder` so I removed `updateLookup` and renamed `updateLayerOrder` to `updateLayer` since that is a more accurate description of what is happening.